exec/java-exec/src/main/java/org/apache/drill/exec/work/fragment/FragmentExecutor.java

```diff
@@ -224,6 +224,7 @@ public void run() {
           ImpersonationUtil.getProcessUserUGI();
 
       queryUserUgi.doAs(new PrivilegedExceptionAction<Void>() {
+        @Override
```

Comment:
Please undo these overrides. Although useful, these are unrelated to the issue. Maybe another ticket?